### ISSUE TITLE
alltt package incompatibility

### ISSUE THREAD

**Issue by mbertucci47** [latex3/tagging-project#131]
The tagging for the `alltt` environment of the alltt package is not ideal. I guess it should be tagged like `verbatim`, so would need special treatment like the latter gets in latex-lab-block.
```tex
\DocumentMetadata
  {
    lang=en-US,
    pdfversion=2.0,
    pdfstandard=ua-2,
    testphase={phase-III,math,title,table,firstaid}
  }
\documentclass{article}
\usepackage{alltt}

\title{alltt tagging test}

\begin{document}

\begin{alltt}
abc \emph{def}
bla ^ _ $
\end{alltt}

\end{document}
```

<img width="218" alt="alltt_struc" src="https://github.com/user-attachments/assets/9b1ffb16-0f77-4de4-b7ad-8d749145c429">

For comparison, replacing `alltt` with `verbatim` above gives

<img width="217" alt="verbatim_struc" src="https://github.com/user-attachments/assets/6dc75b93-bdd1-45c1-9bfb-a9a5e753a45e">

**Comment by FrankMittelbach** [comment]
Agreed, that would need proper support at some later stage, so I guess one should call it currently-incompatible.
